src/main/kotlin/step2/StringCalculator.kt

```diff
@@ -0,0 +1,40 @@
+package step2
+
+class StringCalculator {
+    companion object {
+        private val FORMULA_REGEX = Regex("\\d([+|\\-|*|/]\\d)*")
+        fun calculate(input: String): Int {
+            val expressionStr = input.replace(Regex("\\s+"), "")
+            if (expressionStr.isEmpty() || FORMULA_REGEX.matchEntire(expressionStr) == null) {
+                throw IllegalArgumentException()
+            }
```

Comment:
Exception을 던질 때는 어떤 원인으로 발생한 것인지 메시지를 구체적으로 써주는 것이 좋습니다.
실제로 Exception이 발생했을 때 어떤 원인인지 콘솔창에서 찾고 디버깅하기 쉬워지기 때문이에요 🙂
```suggestion
                throw IllegalArgumentException("구체적인 메시지를 작성해주세요!")
            }
```

Reply:
메시지가 담기는게 좋을 거 같아서 말씀주신대로 수정했습니다